tests/ChunkedEncoding.php

```diff
@@ -15,6 +15,10 @@ public static function chunkedProvider() {
 				"02\r\nab\r\n04\r\nra\nc\r\n06\r\nadabra\r\n0c\r\n\nall we got\n",
 				"abra\ncadabra\nall we got\n"
 			),
+			array(
+				"02;foo=bar;hello=world\r\nab\r\n04;foo=baz\r\nra\nc\r\n06;justfoo\r\nadabra\r\n0c\r\n\nall we got\n",
```

Comment:
Is OWS (optional whitespace) allowed here too? Should add a test for that if so. Ditto if params can take quoted values.